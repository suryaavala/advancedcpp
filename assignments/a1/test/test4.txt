10 sqrt
10.0 sqrt