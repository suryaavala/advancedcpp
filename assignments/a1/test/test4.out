sqrt 10 = 3
sqrt 10.000 = 3.162
